authz/audit/stdout/stdout_logger.go

```diff
@@ -0,0 +1,99 @@
+/*
+ *
+ * Copyright 2023 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// Package stdout defines an stdout audit logger.
+package stdout
+
+import (
+	"encoding/json"
+	"log"
+	"time"
+
+	"google.golang.org/grpc/authz/audit"
+	"google.golang.org/grpc/grpclog"
+)
+
+var grpcLogger = grpclog.Component("authz-audit")
+
+func init() {
+	audit.RegisterLoggerBuilder(&loggerBuilder{})
+}
+
+type event struct {
+	FullMethodName string `json:"fullMethodName"`
+	Principal      string `json:"principal"`
+	PolicyName     string `json:"policyName"`
+	MatchedRule    string `json:"matchedRule"`
+	Authorized     bool   `json:"authorized"`
+	// Timestamp represents time when Log method prints the audit.Event
+	Timestamp string `json:"timestamp"`
+}
+
+// StdoutLogger contains Log method to be invoked to log audit.Event.
+type logger struct {
+}
+
+// Log marshals the audit.Event to json and prints it using log.go
+func (logger *logger) Log(event *audit.Event) {
+	jsonBytes, err := json.Marshal(convertEvent(event))
+	if err != nil {
+		grpcLogger.Errorf("failed to marshal AuditEvent data to JSON: %v", err)
+	}
+	log.Println(string(jsonBytes))
+}
+
+const (
+	stdName = "stdout"
+)
```

Comment:
Since this is used only in one place, I would prefer inline it in the `Name` method.

Reply:
`grpclb` package is not a good example and even there the const is used in more than one place.